src/containers/Article/index.js

```diff
@@ -61,6 +65,14 @@ export class ArticleForm extends Component {
     this.setState({ body: blocks });
   };
 
+  handleTags = (tags) => {
+    const tagArray = [];
+    tags.map((value) => {
```

Comment:
it's a good practice to name your function arguments appropriately. In this case, if I have an array of tags and mapping over it, am expecting to be getting a tag at each iteration, having the name as value is ambiguous. Just for best practices its better doing this

`tags.map((tag) =>{....}`

like in line 71, `value.value` is misleading , 

> object `value` has a property `value`!!!

Reply:
Got it , let me do it